packages/vow/src/watch.js

```diff
@@ -1,15 +1,13 @@
 // @ts-check
-import { M } from '@endo/patterns';
-
-import { getVowPayload, unwrapPromise, basicE } from './vow-utils.js';
+import { getVowPayload, basicE } from './vow-utils.js';
 import { PromiseWatcherI, watchPromiseShim } from './watch-promise.js';
 
 const { apply } = Reflect;
 
 /**
  * @param {typeof watchPromiseShim} watchPromise
  */
-const makeWatchVow =
+const makeRetryAndWatchVow =
   watchPromise =>
   /**
    * @param {any} specimen
```

Comment:
```suggestion
  /**
   * Associate a (usually durable) watcher object with a vow or promise.
   *
   * @param {any} specimen
```
(at which point we can probably go back to `watchVow` rather than `retryAndWatchVow`)

Reply:
I'll call it `watchNextStep`.